LocalMultiplayerAgent/NoOpSessionHostManager.cs

```diff
@@ -27,6 +27,12 @@ public bool LinuxContainersOnWindows
 
         public int StateChangeSequenceNumber { get; }
 
+        public VmAgentSettings VmAgentSettings { get; } = new VmAgentSettings() 
+        { 
+            EnableCrashDumpProcessing = true,
```

Comment:
should we have them as false?

Reply:
My thinking was that we want these settings to all work in LocalMultiplayerAgent as well, but i can see it being worth leaving them false so that we aren't making developers alpha test for us.